chapter3.adoc

```diff
@@ -3,8 +3,12 @@
 == `Smsdid`: Supervisor Domain Identifier and Protection Register
 
 `Smsdid` defines an interface to program the active supervisor domain under
-which a hart is operating. The interface consists of M-mode CSRs `msdcfg` and
-`mmpt`. The SDID programmed via this interface is a local identifier for the
+which a hart is operating and configuration for supervisor domain-related
```

Comment:
"`Smsdid` defines an interface to program the active supervisor domain under which a hart is operating" 

Somehow does not read right to me.

Reply:
How about saying: `Smsdid` defines an interface to program "the protections for" the active supervisor domain under which a hart is operating

Reply:
see updated - thanks